### ISSUE TITLE
tensorflow.js

### ISSUE THREAD

**Issue by MatPoliquin** [divamgupta/stable-diffusion-tensorflow#57]
Anyone tried these models (after exporting them) with tensorflow.js?

**Comment by andreclaudino** [comment]
No, I am planning in using thet in a JS environment. Do you have any results/suggestions that I can use to accelerate my plans?

**Comment by stephenjason89** [comment]
Also interested in this

**Comment by divamgupta** [comment]
https://github.com/mlc-ai/web-stable-diffusion
Check this out

**Comment by stephenjason89** [comment]
@divamgupta thank you for the link
It is really early for this and it only works on chrome canary with this flag at the moment

/Applications/Google\ Chrome\ Canary.app/Contents/MacOS/Google\ Chrome\ Canary --enable-dawn-features=disable_robustness

I hope it gets implemented everywhere soon
